src/senaite/core/browser/static/js/senaite.core.analysisrequest.add.coffee

```diff
@@ -458,7 +458,11 @@ class window.AnalysisRequestAdd
     base_url = window.location.href
     if base_url.search("/portal_factory") >= 0
       return base_url.split("/portal_factory")[0]
-    return base_url.split("/ar_add")[0]
+    url = window.location.origin + window.location.pathname
+    paths = url.split("/")
+    # delete current page path for getting current context
+    paths.pop()
+    return paths.join("/")
```

Comment:
Hi Anton,

can you check if we could simply use `data-view-url` or `data-base-url` from the `body` element instead of that whole dance above?

E.g. `document.body.dataset.baseUrl` or `document.body.dataset.viewUrl`